### PR TITLE
Unicode fix for ItemTable

### PR DESCRIPTION
**Pull request by jlstevens**
This PR aims to address the unicode issues raised in #756.

### CODE REVIEW

File: holoviews/plotting/mpl/tabular.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def pprint_value(self, value):
         elif isinstance(value, str):
             formatted = safe_unicode(value)
         else:
-            formatted = str(value)
```
Comment (philippjfr) [inline holoviews/plotting/mpl/tabular.py]: ``safe_unicode`` will not cast a non-string type to unicode, so if something downstream of this is expecting a string this won't do it.

Comment (jlstevens) [inline holoviews/plotting/mpl/tabular.py]: Right, this change makes the `` elif isinstance(value, str):`` condition pointless. What would you recommend? Simply using ``str()`` results in unicode errors.

Comment (jlstevens) [inline holoviews/plotting/mpl/tabular.py]: ah...I think I should just use...

```
elif isinstance(value, (str, unicode)):
```

I'll try that.

Comment (philippjfr) [inline holoviews/plotting/mpl/tabular.py]: Hmm, changing the previous block with:

```
elif isinstance(value, basestring):
```

where basestring is import from holoviews.core.util, should do it.

**Comment by philippjfr** [comment]
Looks fine, let's see about tests.

**Pull request merged**